.github/workflows/e2e.yaml

```diff
@@ -88,7 +88,9 @@ jobs:
           overwrite: true
 
   e2e-tests:
+    continue-on-error: true
```

Comment:
See the last couple of commits of this PR individually. As ginkgo runs tests in random order I noted that if the new test is executed last I was having some glitches due to elemental-operator container being still in `containerCreation` stage at the time after suite logic to collect logs was executed.

This is to prevent failing all still pending tests if one in the matrix fails. It helps to debug this sort of sporadic issues.